packages/wallets/src/features/cfd/modals/CTraderAddAccountSuccessModal/SuccessIcon/SuccessIcon.scss

```diff
@@ -0,0 +1,8 @@
+.wallets-ctrader-success-icon {
```

Comment:
@hasan-deriv 
seems like we are not importing this scss :thinking_face:
also classname in SuccessIcon.tsx is `wallets-ctrader-success-icon-check` but here classname is `wallets-ctrader-success-icon__check`